migrations/20230712143139_add_vehicle_id_col_to_integration_tbl.sql

```diff
@@ -0,0 +1,19 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+SET search_path = devices_api, public;
+
+alter table user_device_api_integrations
```

Comment:
Let’s not add a column. I could see an argument for adding a `metadata` field. In fact we don’t need to pass this along anymore.